bin/select_targets

```diff
@@ -19,6 +19,7 @@ ap = ArgumentParser()
 ap.add_argument("src", help="Tractor file or root directory with tractor files")
 ap.add_argument("dest", help="Output target selection file")
 ap.add_argument('-v', "--verbose", action='store_true')
+ap.add_argument('-c', "--check", action='store_true')
```

Comment:
Please add a help= string to this argument so that someone running `select_targets --help` will be told what the `--check` option will do.  Otherwise fine to merge.